guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc

```diff
@@ -0,0 +1,89 @@
+[[chap-Documentation-Architecture_Guide-Capsule_Server_Overview]]
+== {SmartProxyServer} Overview
+
+{SmartProxyServer}s provide *content federation* and run *localized services* to discover, provision, control, and configure hosts. You can use {SmartProxies} to extend the {Project} deployment to various geographical locations. This section contains an overview of features that can be enabled on {SmartProxies} as well as their simple classification.
+
+For more information about {SmartProxy} requirements, installation process, and scalability considerations, see link:{BaseURL}installing_capsule_server/[Installing {SmartProxyServer}].
+
+[[sect-Documentation-Architecture_Guide-Capsule_Features]]
+=== {SmartProxy} Features
+
+There are two sets of features provided by {SmartProxyServer}s. You can configure the {SmartProxy} to mirror content from the {ProjectServer}. You can also use the {SmartProxy} to run services required for host management.
+
+Content related features are:
```

Comment:
Content related features, provided by Katello plugin, are: (only upstream, I think it is fair to point this out once again).